package.json

```diff
@@ -28,58 +28,59 @@
     "react": ">=15.0.0"
   },
   "dependencies": {
-    "deep-equal": "1.0.1",
-    "object-assign": "^4.0.1",
+    "deep-equal": "^1.0.1",
```

Comment:
Why are we loosening the rules? we've been burned in the past.  Shouldn't we keep them locked?